frontend/src/App.js

```diff
@@ -1,8 +1,10 @@
 import React, { useState } from "react";
 import axios from "axios";
-import ReactMarkdown from "react-markdown";
 import "./App.css";
-import Ducki from "./assets/ducki.ico";
+import Popup from "./components/popup"
```

Comment:
I like that you added a components folder and are using it to modularize some of the app.

Reply:
Hopefully we can get more of the app inside of the components folder